src/tests/issues.targets

```diff
@@ -2361,6 +2361,9 @@
 
 
     <ItemGroup Condition=" '$(TargetArchitecture)' == 'wasm' " >
+        <ExcludeList Include="$(XunitTestBinBase)/tracing/eventcounter/runtimeeventsource/runtimeeventsource*">
+            <Issue>AppContext switch events are not available on mono Android and wasm</Issue>
+        </ExcludeList>
```

Comment:
Just curious what's special about these two,  e.g. they are available on iOS?

Reply:
Not sure, still investigating. I suspect it doesn't work on iOS either, but we may not run those tests.

Reply:
Yeah -- it looks like the runtime tests are only run on Mono for Android and wasm. I don't even see x64 there, which suprises me.

I'm going to blanket disable this on Mono for now. It wasn't a requirement and I would want to do a lot more work before stating this should work on in any Mono environment.